Formula/haruhi-dl.rb

```diff
@@ -0,0 +1,20 @@
+class HaruhiDl < Formula
+  include Language::Python::Virtualenv
+
+  desc "Fork of youtube-dl, focused on bringing a fast, steady stream of updates"
+  homepage "https://haruhi.download"
+  url "https://files.pythonhosted.org/packages/71/fb/e4e98f70056e9e692ee8222ab708a6e74a7878ea65dfcdaec0b502b80aff/haruhi_dl-2021.3.30.tar.gz"
+  sha256 "3d8a91838bd8a683588c529c6315a960900f5e32e30f5c6051ec836acbb352da"
+  license "LGPL-3.0-or-later"
+
+  depends_on "python@3.9"
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    # History of homebrew-core (video)
+    system "#{bin}/haruhi-dl", "--simulate", "https://www.youtube.com/watch?v=pOtd1cbOP7k"
```

Comment:
Actually, sorry; I forgot this one. Can we check any output here with an `assert_match` or (better) `assert_equal`?

Reply:
fixed